Arithmetic.pp

```diff
@@ -44,12 +44,12 @@
 %token  bracket_  \(
 %token _bracket   \)
 %token  comma     ,
-%token  number    (0|[1-9]\d*)(\.\d+)?([eE][\+\-]?\d+)?
+%token  number    (0|[1-9]\d*)(\.\d+)?([eE][\+\-]?\d+)?\b
```

Comment:
i was wondering what the best solution was ;) I'll move the lines if you prefer :)